core/templates/pages/about-page/about-page.controller.ts

```diff
@@ -76,18 +67,20 @@ angular.module('oppia').directive('aboutPage', [
             ctrl.TAB_ID_ABOUT = 'about';
             ctrl.TAB_ID_FOUNDATION = 'foundation';
             ctrl.TAB_ID_CREDITS = 'credits';
+            ctrl.TAB_ID_LICENSE = 'license';
 
             var hash = WindowRef.nativeWindow.location.hash.slice(1);
-            if (hash === ctrl.TAB_ID_FOUNDATION || hash === 'license') {
-              activateTab(ctrl.TAB_ID_FOUNDATION);
+            if (hash === ctrl.TAB_ID_FOUNDATION || (
+              hash === ctrl.TAB_ID_LICENSE)) {
+              ctrl.activeTabName = ctrl.TAB_ID_FOUNDATION;
             }
 
             if (hash === ctrl.TAB_ID_CREDITS) {
-              activateTab(ctrl.TAB_ID_CREDITS);
+              ctrl.activeTabName = hash;
             }
 
             if (hash === ctrl.TAB_ID_ABOUT) {
-              activateTab(ctrl.TAB_ID_ABOUT);
+              ctrl.activeTabName = hash;
```

Comment:
There are still unnecessary ifs... This can be simplified to:
```typescript
if (hash === ctrl.TAB_ID_LICENSE) {
  ctrl.activeTabName = ctrl.TAB_ID_FOUNDATION;
} else {
  ctrl.activeTabName = hash;
}
```

Reply:
Maybe instead of simple else, you could check if the hash is one of the tabs, preferably by having a list of the allowed tabs and checking if the hash is one of the elements.

Reply:
Done.